### ISSUE TITLE
Unexpected lags

### ISSUE THREAD

**Issue by PiotrGardocki** [SPC-Some-Polish-Coders/PopHead#450]

**Comment by Czapa10** [comment]
> in Profiling64 lags are not visible in the game, but in profiling logs
That's not true. These lags are very noticable in the game and they also appear on Release and Distribution.
